.github/workflows/docker-release.yml

```
@@ -0,0 +1,52 @@
+name: Daily build
+on:
+  schedule:
+    - cron: "0 1 * * *" # Every day at 1:00 AM
+  workflow_dispatch: {}
+jobs:
+  release_image:
+    name: Build and publish docker image
+    runs-on: ubuntu-latest
+    env:
+      IMAGE_REPOSITORY: mongodb/atlas
+      PLATFORMS: linux/amd64,linux/arm64
+    steps:
+      - name: Check out code
+        uses: actions/checkout@v3
+        with:
+          fetch-depth: 0
+      - name: Set date
+        id: set-date
+        run: |
+          DATE=$(date +'%Y-%m-%d')
+          echo DATE=${DATE} >> $GITHUB_ENV
+      - name: 'Get latest tag'
+        id: get-latest-tag
+        run: echo "::set-output name=tag::$(git tag --list --sort=-version:refname "atlascli*" | head -n 1)"
+      - name: Extract version
+        run: |
+          release_tag=${{ steps.get-latest-tag.outputs.tag }}
+          echo "LATEST_VERSION=${release_tag#*/}" >> $GITHUB_ENV
+      - name: Set up QEMU
+        uses: docker/setup-qemu-action@v2
+        with:
+          platforms: ${{ env.PLATFORMS }}
+      - name: Set up Docker Buildx
+        uses: docker/setup-buildx-action@v2
+        with:
+          platforms: ${{ env.PLATFORMS }}
+      - name: Login to Docker Hub
+        uses: docker/login-action@v2
+        with:
+          username: "${{ secrets.DOCKERHUB_USER }}"
+          password: "${{ secrets.DOCKERHUB_SECRET }}"
+      - name: Build and push container to Dockerhub
+        uses: docker/build-push-action@v4
+        with:
+          context: .
+          platforms: ${{ env.PLATFORMS }}
+          tags: ${{ env.IMAGE_REPOSITORY }}:latest ,
+            ${{ env.IMAGE_REPOSITORY }}:${{ env.LATEST_VERSION }} ,
+            ${{ env.IMAGE_REPOSITORY }}:${{ env.LATEST_VERSION }}-${{ env.DATE }}
+          file: Dockerfile
+          push: true
```

```suggestion
          push: false
```

I would recommend testing job without pushing. If everything works and we are happy then we can have push enabled.